dev/ProjectReunion_DLL/pch.h

```diff
@@ -9,6 +13,7 @@
 #include <wil/token_helpers.h>
 #include <wil/resource.h>
 #include <wil/result_macros.h>
+#include <shellapi.h>
 #include <appmodel.h>
```

Comment:
shellapi.h + appmodel.h should come before WIL headers

Good header order is more generic to specific e.g.
* Windows
* Standard C++
* WIL
* ProjectReunion